spatial_db_msgs/ObjectDescription[] descriptions
---
int32[] ids
